cmd/nodecmd/create.go

```diff
@@ -137,8 +137,15 @@ will apply to all nodes in the cluster`,
 func handlePostRun(_ *cobra.Command, _ []string) {}
 
 func preCreateChecks(clusterName string) error {
-	if !flags.EnsureMutuallyExclusive([]bool{useLatestAvalanchegoReleaseVersion, useLatestAvalanchegoPreReleaseVersion, useAvalanchegoVersionFromSubnet != "", useCustomAvalanchegoVersion != ""}) {
-		return fmt.Errorf("latest avalanchego released version, latest avalanchego pre-released version, custom avalanchego version and avalanchego version based on given subnet, are mutually exclusive options")
+	if useCustomAvalanchegoVersion != "" || useAvalanchegoVersionFromSubnet != "" {
```

Comment:
I am confused with this one. Why do we want to set flags to false `useLatestAvalanchegoReleaseVersion` and `useLatestAvalanchegoPreReleaseVersion` as user input them and we silently change them to something else based on other flags VS how it was before and we just return error to user if we tries to use mutually exclusive flags

Reply:
I guess I understand now why, but we should display user a message explaining why we ignore their input.